packages/portal/src/components/home/HomePage.vue

```diff
@@ -82,31 +83,16 @@
       pageMeta() {
         return {
           title: this.$t('homePage.title', { digital: this.$t('homePage.titleDigital') }),
-          description: this.pageSubHeadline,
+          description: this.$t('homePage.subHeadline'),
+          // TODO: make this apply even if the homePage content type isn't requested,
+          //       e.g. when a landing page override has been requested at / instead
           ogType: 'website',
-          ogImage: this.headMetaOgImage
+          ogImage: this.$contentful.assets.optimisedSrc(this.socialMediaImage, { w: 1200, h: 630, fit: 'fill' })
         };
-      },
-
-      pageSubHeadline() {
-        return this.$t('homePage.subHeadline');
-      },
-
-      headMetaOgImage() {
-        const image = this.socialMediaImage ? this.socialMediaImage : this.backgroundImage?.image;
```

Comment:
if we're not using this fallback because the homepage _should_ have a distinct social media image, shouldn't we also make the social media image a required field?

Reply:
Background image isn't required either, so, no.